.vsts-ci.yml

```diff
@@ -268,13 +268,21 @@ stages:
     parameters:
       agentOs: Darwin
       pool: 
-        name: Hosted macOS
+        vmImage: 'macOS-10.15'
       timeoutInMinutes: 180
       strategy:
         matrix:
-          Build_Release:
+          Build_Release_x64:
             _BuildConfig: Release
+            _RuntimeIdentifier: ''
```

Comment:
should this also specifi the rid for consistency?

Reply:
Empty string is used in many places above when it matches the machines runtime.  So this was relatively consistent.